prisma/migrations/20240309184005_database_schema_v1/migration.sql

```diff
@@ -2,7 +2,8 @@
 CREATE TABLE "user" (
     "id" SERIAL NOT NULL,
     "username" TEXT NOT NULL,
```

Comment:
TEXT seems a bit too much for a username
Why not
```suggestion
    "username" VARCHAR(20) NOT NULL,
```